src/cat-data-service/src/service/api/health/started_get.rs

```diff
@@ -0,0 +1,42 @@
+//! Implementation of the GET /health/started endpoint
+
+use crate::service::generic::responses::resp_2xx::NoContent;
+use crate::service::generic::responses::resp_5xx::{ServerError, ServiceUnavailable};
+
+use poem_extensions::response;
+use poem_extensions::UniResponse::T204;
+
+pub(crate) type AllResponses = response! {
```

Comment:
I think we can move type allias to the root of the `health` module. So we can remove duplication of the code

Reply:
It's not guaranteed to be universal per endpoint; these are the responses generated by this endpoint only.
The fact they are currently all the same is a coincidence.

I would rather have a pattern where the endpoint defines everything about it and not have special cases.